src/main/java/org/fundacionjala/at15/pokemon/Gym.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.at15.pokemon;
+
+import java.util.*;
+
+public class Gym {
+    private final int hitpoints = 100;
+    private Pokemon gymLeaderPokemon = new Pokemon(hitpoints, "gymLeaderPokemon");
+    private Trainer gymLeader = new Trainer(gymLeaderPokemon, "gymLeader");
+    private Pokemon pokemon1 = new Pokemon(hitpoints, "pokemon1");
+    private Trainer lowerRankingMember1 = new Trainer(pokemon1, "lowerRankingMember1");
+    private Pokemon pokemon2 = new Pokemon(hitpoints, "pokemon2");
+    private Trainer lowerRankingMember2 = new Trainer(pokemon2, "lowerRankingMember2");
+
+    private ArrayList<PokemonTeam> gymPokemonTeams = new ArrayList<PokemonTeam>();
+    private ArrayList<Trainer> trainers = new ArrayList<Trainer>();
+
+    public Gym() {
+        trainers.add(lowerRankingMember2);
```

Comment:
To avoid create every trainer of the gym, maybe we could open another ticket to create the clas TrainersGym or something like that and the constructor Gym() will call to that class, Gym(TrainersGym customGym), don't you think?

Reply:
It could be, but maybe for now it could work this way since the gym class does not have another purpose